polaris-migrator/src/migrations/v12-styles-replace-custom-property-shadow/transform.ts

```diff
@@ -19,9 +19,6 @@ const replacementMaps = {
     '--p-shadow-xl': '--p-shadow-500',
     '--p-shadow-2xl': '--p-shadow-600',
     '--p-shadow-bevel-experimental': '--p-shadow-bevel-100',
-    '--p-shadow-card-sm-experimental': '--p-shadow-100',
-    '--p-shadow-card-md-experimental': '--p-shadow-200',
-    '--p-shadow-card-lg-experimental': '--p-shadow-300',
```

Comment:
alternatively I could've mapped these to hardcoded values but best practices would be to use the `Card` component, or compose something similar to our `ShadowBevel` component (which isn't exported). Asking consumers to apply pseduo elements may get complicated though, so I've just suggested they use `Card` for all previous experimental card levels